crates/testing/tests/tests_1/quorum_vote_task.rs

```diff
@@ -23,26 +23,31 @@ async fn test_quorum_vote_task_success() {
     let quorum_membership = handle.hotshot.memberships.quorum_membership.clone();
     let da_membership = handle.hotshot.memberships.da_membership.clone();
 
-
     let mut generator = TestViewGenerator::generate(quorum_membership.clone(), da_membership);
 
-    generator.next();
-    let view = generator.current_view.clone().unwrap();
+    let mut proposals = Vec::new();
+    let mut leaves = Vec::new();
+    let mut dacs = Vec::new();
+    let mut vids = Vec::new();
+    for view in (&mut generator).take(2) {
+        proposals.push(view.quorum_proposal.clone());
+        leaves.push(view.leaf.clone());
+        dacs.push(view.da_certificate.clone());
+        vids.push(view.vid_proposal.clone());
+    }
 
     // Send the quorum proposal, DAC, and VID disperse data, in which case a dummy vote can be
     // formed and the view number will be updated.
     let view_success = TestScriptStage {
         inputs: vec![
-            QuorumProposalRecv(view.quorum_proposal.clone(), view.leader_public_key),
-            DACertificateRecv(view.da_certificate.clone()),
-            VIDShareRecv(view.vid_proposal.0[0].clone()),
+            QuorumProposalValidated(proposals[1].data.clone(), leaves[0].clone()),
+            DACertificateRecv(dacs[1].clone()),
+            VIDShareRecv(vids[1].0[0].clone()),
         ],
         outputs: vec![
-            exact(ViewChange(ViewNumber::new(2))),
```

Comment:
Probably unrelated to this PR, but I'm wondering why we no longer have `ViewChange` here and the other tests in this file. I think the vote task still needs this event to call `PollForVIDDisperse`.